src/components/molecules/content/blog-elements/BlogHeader.svelte

```
@@ -0,0 +1,24 @@
+
+
+<header class="sticky top-0 z-50 mb-12 flex w-full items-center justify-between bg-white px-6 py-4">
+	<div class="flex items-center gap-8">
+		<a href="/home" class="flex items-center gap-3">
```

ESLint

[error] Found a link with a url that isn't resolved.

(svelte/no-navigation-without-resolve)